### DISCUSSION TITLE
Open PDF in presentation mode (full screen) then change page every 1 min... when at the end, restart

### DISCUSSION THREAD

**Discussion by Fujoubou** [sumatrapdfreader/sumatrapdf#4081] [Q&A]
Whould it be possible to do this ?
Open PDF in presentation mode (full screen) then change page every 1 min... when at the end, restart

**Comment by GitHubRulesOK** [comment]
yes
 but you need to write the series of instructions EITHER in CMD or DDE or any other you know you can use to talk to either of those.

so the simpler one is in outline terms below
adjust to suit your file locations number of pages and timing  I used this (without full screen) to answer my own GIF animation question #1343 

HOWEVER BEWARE once it is running it is hard to stop unless you kill the command task in the taskbar

Use ESC to exit full screen
```
REM switch OFF "Remember files" in Options otherwise it may not go full screen
REM since on first call it will remember the last page position.

REM edit the path to run your own copy
set "SumatraPDF=C:\program files\sumatrapdf\sumatrapdf.exe"
set "PDFfile=c:\folder\My Presentation.pdf"

Rem start SumatraPDF fullscreen with firstpage (will be shown twice unless you call a seperate start page or image)
start "Showtime" "%SumatraPDF%" -fullscreen -page 1 "%PDFfile%"

:loop
for /L %%t in (1,1,3) do timeout 4 & "%SumatraPDF%" -page %%t "%PDFfile%"
REM Beware using this one makes it much harder to stop 
REM for /L %%t in (1,1,3) do timeout 4 & start "Showtime" "%SumatraPDF%" -page %%t "%PDFfile%"
goto loop

REM Should never get here
```

however that will try to show the timeout on screen so it would possibly needs be running minimised
